app/browser/api/ledger.js

```diff
@@ -233,7 +233,7 @@ const getPublisherTimestamp = (updateList) => {
   client.publisherTimestamp(publisherTimestampFn)
 }
 
-const publisherTimestampCallback = (updateList, err, result) => {
+const publisherTimestampCallback = (err, result, updateList) => {
```

Comment:
This will not work AFAIK, because we bind `updateList` to this function and bind is first param and then params that are added. Is this working correctly?